src/bun.js/bindings/webcore/FetchHeaders.cpp

```diff
@@ -237,7 +239,12 @@ ExceptionOr<void> FetchHeaders::set(const String& name, const String& value)
         return {};
 
     ++m_updateCounter;
-    m_headers.set(name, normalizedValue);
+    if (equalIgnoringASCIICase(name, setCookieLowercaseString)) {
```

Comment:
Please see the code in `HTTPHeaderMap::set`

This is unnecessary duplicate work that will make the Headers object slower

https://github.com/oven-sh/bun/blob/0b183beb51367004795d8a431eb06bb2fa4f8250/src/bun.js/bindings/webcore/HTTPHeaderMap.cpp#L112-L115

We can use the header name from here ^